### PR TITLE
mainblock added

### PR DESCRIPTION
**Pull request by EyR1oN**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by Tatiana2424 — APPROVED** [review]

**Review by PingvinAustr — APPROVED** [review]

File: src/features/MainBlock/MainBlock.styles.scss [hunk 1]
```diff
@@ -3,14 +3,49 @@
...
+.mainContainer {
+  width: pxToRem(1300px);
+  position: relative;
+}
+
+.breadcrumbContainer {
+  @include mut.with-font($font-weight: 300, $font-size:12px);
+  position: absolute;
+  top: pxToRem(12px);
+  right: pxToRem(50px);
```
Comment (AleXLaeR) [inline src/features/MainBlock/MainBlock.styles.scss:20]: Consider **positioned-as** mixin.

File: src/features/MainBlock/MainBlock.styles.scss [hunk 2]
```diff
@@ -3,14 +3,49 @@
 @use "@assets/sass/variables/_variables.fonts.scss" as ft;
 @use "@assets/sass/_utils.functions.scss" as *;
 
-.header {
-  background-color: c.$pure-white-color;
+.mainStreetcodeBlock {
+  @include mut.flex-centered();
+  background: url('@assets/images/background.png');
```
Comment (AleXLaeR) [inline src/features/MainBlock/MainBlock.styles.scss:8]: Consider **bg-image** mixin.

File: src/features/MainBlock/MainBlock.styles.scss [hunk 3]
```diff
@@ -3,14 +3,49 @@
...
+
+:where(.css-dev-only-do-not-override-26rdvq).ant-breadcrumb a {
+  color: c.$dark-red-color;
+}
+
+.separator {
+  display: inline-block;
+  @include mut.sized(4px, 4px);
+  background-color: c.$dark-red-color;
+  margin-bottom: pxToRem(2px);;
```
Comment (AleXLaeR) [inline src/features/MainBlock/MainBlock.styles.scss:35]: Unnecessary semicolon.

File: src/features/MainBlock/MainBlock.styles.scss [hunk 4]
```diff
@@ -75,30 +104,55 @@
...
-  @include mut.rem-padded(16px, 0, 16px, 10px);
+  @include mut.rem-padded(13px, 0, 13px, 3px);
   @include mut.flexed($align-items: flex-start, $wrap: wrap, $gap: 5px);
 }
 
 .tagItem {
-  padding: pxToRem(9px) pxToRem(12px);
+  @include mut.rem-padded(8px, 10px, 8px, 10px);
   color: c.$accented-red-color;
+  height: 38px;
```
Comment (AleXLaeR) [inline src/features/MainBlock/MainBlock.styles.scss:114]: Please convert 'px' into 'rems' at all times.

File: src/features/MainBlock/MainBlock.styles.scss [hunk 5]
```diff
@@ -75,30 +104,55 @@
...
+  border-color: c.$accented-red-color;
+  @include mut.full-rounded(14px);
+  background-color: c.$accented-red-color;
+}
+
+.animateFigureBtn {
+  @include mut.sized(218px, 73px);
+  @include mut.with-font($font-weight: 500);
+  border-color: c.$accented-red-color;
+  border-width: pxToRem(3px);
```
Comment (AleXLaeR) [inline src/features/MainBlock/MainBlock.styles.scss:149]: Never set border values' units to any other value than px.

**Review by AleXLaeR — CHANGES_REQUESTED** [review]

**Review by AleXLaeR — APPROVED** [review]

**Pull request merged**